lib/elixir/src/elixir_clauses.erl

```diff
@@ -184,11 +184,16 @@ expand_with_else(Meta, Opts, E, HasMatch) ->
   RaiseError = fun(Key) ->
     form_error(Meta, ?key(E, file), ?MODULE, {duplicated_clauses, 'try', Key})
   end,
+  CatchBeforeRescueWarn = fun() ->
+    Message = "\"catch\" should always come after \"rescue\" in a try/do block",
```

Comment:
"try/do" reads like two possible alternatives, I think "in try" would read better.

Reply:
ok